drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractScenarioSimulationCommand.java

```diff
@@ -184,4 +185,20 @@ protected ScenarioGridColumn getScenarioGridColumnLocal(ScenarioSimulationBuilde
         context.getScenarioGridPanel().select();
         return CommandResultBuilder.SUCCESS;
     }
+
+    public void commonInternalExecution(ScenarioSimulationContext.Status status, ScenarioSimulationContext context, int index) {
```

Comment:
Please create a new `AbstractInsertColumnCommand` instead of move this method here and maybe change the name to something like `commonInsertColumn`